target/test-classes/Feature/Login.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Validate the login functionality
+
+Scenario: Validate login with valid credentials
```

Comment:
For the scenario involving invalid credentials, we need to check both valid and invalid credentials

Reply:
Noted. I refreshed the concepts that's why I didn't go for the multiple scenarios. Let me rectify it from upcoming works

Reply:
As mentioned I refreshed the concepts that's why I didn't go for the multiple scenarios. Let me rectify it from upcoming works